webpack.config.js

```diff
@@ -12,6 +12,8 @@ var autoprefixer = require('autoprefixer');
 var postcssVars = require('postcss-simple-vars');
 var postcssImport = require('postcss-import');
 
+const BASE_PATH = process.env.BASE_PATH || '/';
```

Comment:
Judging by the way it's used, it looks like this might be more convenient / descriptive:
```suggestion
const STATIC_PATH = process.env.STATIC_PATH || '/static';
```

I defer to {@chrisg, @paulkaplan, @rschamp}'s opinion though ;)